imagetest/test_suites/storage_perf/startupscripts/install_fio.sh

```diff
@@ -0,0 +1,16 @@
+if [[ -f /usr/bin/apt ]]; then
+	sudo apt -y update && sudo apt -y install fio
+fi
+
+if [[ -f /bin/yum ]]; then
+	sudo yum -y install fio
+fi
+
+if [[ -f /usr/bin/zypper ]]; then
+	sudo zypper --non-interactive install fio
+fi
+
+if [[ -f /bin/dnf ]]; then
```

Comment:
A bit inconsequential but put the dnf before yum (yum in EL8/9 exists as a symlink to dnf).

Reply:
done. also changed it in the golang code to install this, if it becomes necessary again due to script order behavior